hrjobcontract/CRM/Hrjobcontract/Upgrader.php

```diff
@@ -511,6 +511,8 @@ public function upgradeBundle() {
     $this->upgrade_1029();
     $this->upgrade_1030();
     $this->upgrade_1032();
+    $this->upgrade_1033();
```

Comment:
Do you think we could use traits like in other extensions? I know we already have function here, but going forward it might be nicer to use traits for all of them. 

It probably doesn't make sense to have both traits and keep existing functions in the upgrader class, so might be better to make a ticket to move all upgraders to using traits instead of half-doing it in this ticket.

Reply:
It's possible to use them here, but that would require transforming all the existing upgraders to traits. This extension is already a mess and using traits only for new upgraders would only make it messier. 

Given there are plans to refactor the Job Contract extension in the future, it better to wait for that until we start using traits here